Database.py

```diff
@@ -0,0 +1,237 @@
+import sqlite3
+import pandas as pd
+import os
+import requests
+import re
+import json
+import argparse
+
+
+def create_database():
+    """Creates the premier_league.db."""
+    db_path = "premier_league.db"
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+
+    # Create match_status table
+    cursor.execute('''
+    CREATE TABLE IF NOT EXISTS match_status (
+        Name TEXT NOT NULL,
+        Played INT,
+        Wins INT,
+        Draws INT,
+        Losses INT,
+        GF INT,
+        GA INT,
+        GoalDiff INT,
+        Pts INT,
+        season TEXT NOT NULL,
+        status_type TEXT NOT NULL,
+        PRIMARY KEY (Name, status_type, season)
+    )
+    ''')
+
+    # Create Stadiums table
+    cursor.execute('''
+    CREATE TABLE IF NOT EXISTS Stadiums (
+        Team TEXT PRIMARY KEY,
+        FDCOUK TEXT,
+        City TEXT,
+        Stadium TEXT,
+        Capacity INT,
+        Latitude REAL,
+        Longitude REAL,
+        Country TEXT
+    )
+    ''')
+
+    # Create Teams table
+    cursor.execute('''
+    CREATE TABLE IF NOT EXISTS Teams (
+        id INTEGER PRIMARY KEY AUTOINCREMENT,
+        Team TEXT UNIQUE
+    )
+    ''')
+
+    conn.commit()
+    conn.close()
+    print("Database and tables created.")
+
+
+def load_games_table():
+    """Loads all data from Games_season19-20.csv into the Games table."""
+    conn = sqlite3.connect("premier_league.db")
+
+    # Read the CSV file
+    games_df = pd.read_csv('Games_season19-20.csv')

```

Comment:
En í ykkar tilfelli þá er þessi skrá í möppu sem heitir CSV:
```suggestion
    games_df = pd.read_csv('CSV/Games_season19-20.csv')

```